### PR TITLE
Add CHANGELOG.md to promptflow-tools and update promptflow-tools version to 1.0.0

### PR DESCRIPTION
**Pull request by chenslucky**
# Description

Add CHANGELOG.md to promptflow-tools and update promptflow-tools version to 1.0.0.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by DaweiCai — DISMISSED** [review]

File: src/promptflow-tools/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 0.1.0b16 (Upcoming)
```
Comment (chjinche) [inline src/promptflow-tools/CHANGELOG.md:3]: 1.0.0 (specific date)

File: src/promptflow-tools/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Features Added
+- Support openai 1.x for promptflow-tools
+- Rename api_base to base_url for openai connection mapping
```
Comment (chjinche) [inline src/promptflow-tools/CHANGELOG.md:7]: Remove this

File: src/promptflow-tools/setup.py [hunk 3]
```diff
@@ -28,12 +28,15 @@ def parse_requirements(file_name: str) -> List[str]:
...
 
+with open("CHANGELOG.md", encoding="utf-8") as f:
+    changelog = f.read()
+
 setup(
     name=PACKAGE_NAME,
     version=version,
     description="Prompt flow built-in tools",
     long_description_content_type="text/markdown",
-    long_description=readme,
```
Comment (chjinche) [inline src/promptflow-tools/setup.py:36]: long_description_content_type="text/markdown",

File: src/promptflow-tools/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 0.1.0b16 (Upcoming)
+
+### Features Added
+- Support openai 1.x for promptflow-tools
```
Comment (chjinche) [inline src/promptflow-tools/CHANGELOG.md:6]: Support OpenAI 1.x in promptflow-tools

File: src/promptflow-tools/CHANGELOG.md (hunk 2 above)
Comment (chjinche) [reply src/promptflow-tools/CHANGELOG.md:7]: Add a new feature 
- `New tool 'OpenAI GPT-4V'`

File: src/promptflow-tools/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0 (2023.11.30)
+
+### Features Added
+- Support openai 1.x in promptflow-tools
+- Add new tool `OpenAI GPT-4V`
```
Comment (chjinche) [inline src/promptflow-tools/CHANGELOG.md:7]: "OpenAI GPT-4V"

**Review by chjinche — APPROVED** [review]

**Review by chjinche — APPROVED** [review]

**Review by dorisjoy — APPROVED** [review]

**Pull request merged**